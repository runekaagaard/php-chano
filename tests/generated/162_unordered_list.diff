001- <li>item 1
001+ Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/162_unordered_list.php on line 23
002- 	<ul>
003- 		<li>item 1.1</li>
004- 		<li>item 1.2</li>
005- 	</ul>
006- 	</li>