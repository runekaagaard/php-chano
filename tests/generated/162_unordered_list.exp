<li>item 1
	<ul>
		<li>item 1.1</li>
		<li>item 1.2</li>
	</ul>
	</li>