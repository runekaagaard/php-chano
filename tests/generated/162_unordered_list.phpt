--TEST--
A generated testfile for the "unordered_list" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => array (
  0 => 'item 1',
  1 => 
  array (
    0 => 
    array (
      0 => 'item 1.1',
      1 => 
      array (
      ),
    ),
    1 => 
    array (
      0 => 'item 1.2',
      1 => 
      array (
      ),
    ),
  ),
))));
foreach ($items as $i) echo $i->input->unordered_list();
--EXPECT--
	<li>item 1
	<ul>
		<li>item 1.1</li>
		<li>item 1.2</li>
	</ul>
	</li>