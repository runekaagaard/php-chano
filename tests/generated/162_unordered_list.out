Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/162_unordered_list.php on line 23